up-discovery-android/service/src/main/java/org/eclipse/uprotocol/core/udiscovery/UDiscoveryService.java

```diff
@@ -0,0 +1,447 @@
+            responseFuture.complete(packToAny(response));
+        }
+    }
+
+    private void executeUpdateNode(@NonNull UMessage requestEvent, @NonNull CompletableFuture<UPayload> future) {
+        Log.d(LOG_TAG, join(Key.EVENT, "executeUpdateNode"));
+        try {
+            checkArgument(mDatabaseInitialized.get(), UCode.FAILED_PRECONDITION, DATABASE_NOT_INITIALIZED);
+            //Log.d(LOG_TAG, Key.EVENT, "CE1 received for UpdateNode", Key.REQUEST, stringify(requestEvent));
+            future.complete(mRpcHandler.processLDSUpdateNode(requestEvent));
+        } catch (Exception e) {
+            UStatus status = errorStatus(LOG_TAG, "executeUpdateNode", toStatus(e));
+            future.complete(packToAny(status));
+        }
+    }
+
+    private void executeFindNodesProperty(@NonNull UMessage requestEvent,
+                                          @NonNull CompletableFuture<UPayload> future) {
+        try {
+            checkArgument(mDatabaseInitialized.get(), UCode.FAILED_PRECONDITION, DATABASE_NOT_INITIALIZED);
+            future.complete(mRpcHandler.processFindNodeProperties(requestEvent));
+        } catch (Exception e) {
+            UStatus status = errorStatus(LOG_TAG, "executeFindNodesProperty", toStatus(e));
+            FindNodePropertiesResponse response = FindNodePropertiesResponse.newBuilder().setStatus(status).build();
+            future.complete(packToAny(response));
+        }
+    }
+
+    private void executeAddNodes(@NonNull UMessage requestEvent, @NonNull CompletableFuture<UPayload> future) {
+        Log.d(LOG_TAG, join(Key.EVENT, "executeAddNodes"));
+        try {
+            checkArgument(mDatabaseInitialized.get(), UCode.FAILED_PRECONDITION, DATABASE_NOT_INITIALIZED);
+            //Log.d(LOG_TAG, Key.EVENT, "CE1 received for AddNodes", Key.REQUEST, stringify(requestEvent));
+            future.complete(mRpcHandler.processAddNodesLDS(requestEvent));
+        } catch (Exception e) {
+            UStatus status = errorStatus(LOG_TAG, "executeAddNodes", toStatus(e));
+            future.complete(packToAny(status));
+        }
+    }
+
+    private void executeDeleteNodes(@NonNull UMessage requestEvent, @NonNull CompletableFuture<UPayload> future) {
+        try {
+            checkArgument(mDatabaseInitialized.get(), UCode.FAILED_PRECONDITION, DATABASE_NOT_INITIALIZED);
+            //Log.i(LOG_TAG, Key.EVENT, "CE1 received for DeleteNodes", Key.REQUEST, stringify(requestEvent));
+            future.complete(mRpcHandler.processDeleteNodes(requestEvent));
+        } catch (Exception e) {
+            UStatus status = errorStatus(LOG_TAG, "executeDeleteNodes", toStatus(e));
+            future.complete(packToAny(status));
+        }
+    }
+
+    private void executeUpdateProperty(@NonNull UMessage requestEvent,
+                                       @NonNull CompletableFuture<UPayload> future) {
+        Log.d(LOG_TAG, join(Key.EVENT, "executeUpdateProperty"));
```

Comment:
Not consistent, some handling methods are printing debug log like this others not

Reply:
This some thing team has to decide. I Just integrate API eclipse proto

Reply:
Sure, we can implement.